WasatchNET/SPISpectrometer.cs

```diff
@@ -4,6 +4,7 @@
 using System.Linq;
 using LibUsbDotNet.Main;
 using MPSSELight;
+using System.Runtime.Remoting.Messaging;

```

Comment:
I am intrigued.

Reply:
I genuinely have no idea why this package is in there